plugins/headless/hu.bme.mit.gamma.headless.api/docs/headless-gamma-document.md

```diff
@@ -120,15 +120,19 @@ This means that the compiler compliance level is set too high. Open the Eclipse
 
 ## Setting up the Docker container
 
- Make sure that Docker is installed on the computer in use. The following tutorial was used to install Docker on Ubuntu: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04
+ Make sure that Docker is installed on the computer in use. The following tutorial was used to install Docker on Ubuntu: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04. Note that the tutorial below focuses on the creation of the docker image, and then running a container from that image. For this reason, it ignores the official Headless Gamma image that can be found here: https://hub.docker.com/repository/docker/ftsrggamma/headless-gamma. It is highly recommended to use the official image. Further information on how to use this image can be found in the Headless Gamma README.
 
  1. Create a working folder, and copy the Dockerfile inside.
- 2. Inside the working folder, create the following folders: "eclipse",  "server", "uppaal", "theta".
- 3. Export the Headless Gamma API inside the "eclipse" folder.
- 5. Copy the "OpenApiGammaWrapper" folder, which contains the webserver, inside the "server" folder.
- 6. Copy the contents of the UPPAAL folder installed on your computer inside the "uppaal" folder.
- 7. Copy the `get-theta.sh` file inside the "theta" folder. This file can be found inside the `gamma/plugins/xsts/theta-bin` folder. 
- 8. Open a terminal and build the docker image with the following command: `docker build -t gamma <path to working folder>` (the working folder is the folder created in the first step) . This will create a docker image named "gamma".
- 9. In a terminal, run a docker container from the gamma image using the `docker run -it -p 8080:8080 --name gamma_container gamma:latest` command. This command will bind the localhost:8080 address of the host machine to the Docker container, forwarding commands to the webserver running inside. The container is named "gamma_container", and starts in interactive mode, allowing for CLI access. The last parameter is the image, which is `gamma:latest`, which indicates that the latest Gamma image build is used.
+ 2. Inside the working folder, create the following folders: "eclipse",  "server", "uppaal", "theta", "project".
```

Comment:
Here and later too: when referring to folders and folder names, you could use the following style `eclipse` instead of "eclipse", to make it more transparent.